libs/blocks/table/table.css

```diff
@@ -590,14 +595,9 @@ header.global-navigation {
   }
 }
 
-@media (max-width: 400px) {
+@media (max-width: 430px) {
```

Comment:
Not blocking the PR, but I'm not particularly happy with having non-standard breakpoints. @overmyheadandbody wdyt?